.github/pull_request_template.md

```diff
@@ -1,46 +1,87 @@
-## Cover letter
+<!--
+
+See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md##pull-request-body
+for more details and examples of what is expected in a PR body.
+
+-->
 
-Describe in plain language the motivation (bug, feature, etc.) behind the change in this PR and how the included commits address it.
+## Cover Letter
 
-<!-- Use the GitHub keyword `Fixes` to link to bug(s) this PR will fix. -->
-Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...
+<!--
+
+Content in this section is REQUIRED.
 
-## Backport Required
+Describe, in plain language, the motivation behind the change (bug fix,
+feature, improvement) in this PR and how the included commits address it.
 
-<!-- Specify which branches this should be backported to, e.g.: -->
-- [ ] not a bug fix
-- [ ] issue does not exist in previous branches
-- [ ] papercut/not impactful enough to backport
-- [ ] v22.2.x
-- [ ] v22.1.x
-- [ ] v21.11.x
+Add the GitHub keyword `Fixes` to link to bug(s) this PR will fix, e.g.
 
-## UX changes
+  Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...
 
-Describe in plain language how this PR affects an end-user. What topic flags, configuration flags, command line flags, deprecation policies etc are added/changed.
+If this PR is a backport, link to the original with `Backport of PR`, e.g.
 
-<!-- don't ship user breaking changes. Ping PMs for help with user visible changes  -->
+  Backport of PR #PR-NUMBER
+
+-->
+
+## Backports Required
 
-## Release notes
 <!--
 
-If this PR does not need to be included in the release notes, then
-simply have a bullet point for `none` directly under the `Release notes`
-section, e.g.
+Content in this section is REQUIRED if this is a PR to `dev` branch.
+
+If this PR needs backport, then list the branches to backport to, e.g.
+
+  - [ ] v22.2.x
+  - [ ] v22.1.x
+  - [ ] v21.11.x
+
+If this PR does not need backport, then list `none`, e.g.
+
+  * none
+
+-->
+
+## UX Changes
+
+<!--
+
+Content in this section is OPTIONAL.
+
+Describe, in plain language, how this PR affects an end-user. Explain
+topic flags, configuration flags, command line flags, deprecation
+policies, etc. that are added or modified. Don't ship user breaking
+changes. Ask the @redpanda-data/product team if you need help with user
+visible changes.
+
+-->
+
+## Release Notes
+
+<!--
+
+Content in this section is REQUIRED if the PR is not a backport PR.
```

Comment:
Not required in all cases.  e.g. we frequently commit refactors, test fixes, etc.

Reply:
Oh, you probably mean content is required in the string sense (but that content may be "* none").  Would be good to clarify that, otherwise folks might get the impression they're supposed to write release notes for absolutely everything

Reply:
i've changed the line to:
> Adding a sub-section or `none` is REQUIRED if the PR is not a backport PR.